### ISSUE TITLE
qwen微调后推理慢的问题

### ISSUE THREAD

**Issue by wangschang** [yangjianxin1/Firefly#147]
正常推理在10s以内返回
微调后25-30s之间，请问是什么参数有影响吗

微调参数：
{
    "output_dir": "output/firefly-qwen-7b_0913",
    "model_name_or_path": "xxxx",
    "train_file": "/root/ptuning/data0912_outputjson.json",
    "num_train_epochs": 8,
    "per_device_train_batch_size": 6,
    "gradient_accumulation_steps": 2,
    "learning_rate": 2e-4,
    "max_seq_length": 1024,
    "logging_steps": 100,
    "save_steps": 3258,
    "save_total_limit": 1,
    "lr_scheduler_type": "constant_with_warmup",
    "warmup_steps": 3000,
    "lora_rank": 64,
    "lora_alpha": 16,
    "lora_dropout": 0.05,
    "gradient_checkpointing": true,
    "disable_tqdm": false,
    "optim": "paged_adamw_32bit",
    "seed": 42,
    "fp16": true,
    "report_to": "tensorboard",
    "dataloader_num_workers": 0,
    "save_strategy": "steps",
    "weight_decay": 0,
    "max_grad_norm": 0.3,
    "remove_unused_columns": false
}
配置参数：
"use_flash_attn": false, 因为报错改的这个 不知道是否有影响？
